### ISSUE TITLE
`onyo git commit` without `-m "message"` crashes

### ISSUE THREAD

**Issue by TobiasKadelka** [psyinfra/onyo#84]
`onyo git commit -m "commit these changes"` works as expected, but `onyo git commit` (should open editor for commit message) crashes.

**Comment by aghisla** [comment]
Both with and without -m, I get an error from onyo (source code from 2022-08-, but no crashes nor commits:

I edited one of my templates, to have something to commit.

```
(onyo) aghisla@juvuzela in ~/source/onyo-testrepo on git:master!
❱ onyo git commit                                                           1 !
ERROR:onyo:
Onyo expects a clean onyo working tree before running commands. Please commit or .gitignore all changes and check the syntax of asset files.

The Onyo working tree is not clean.
Changes not staged for commit: 
	 .onyo/templates/laptop.template
Please commit all changes or add new files to .gitignore
(onyo) aghisla@juvuzela in ~/source/onyo-testrepo on git:master!
❱ onyo git commit -m"Commented the vendor model type keys in laptop template, duplicates of the asset creation questions."
ERROR:onyo:
Onyo expects a clean onyo working tree before running commands. Please commit or .gitignore all changes and check the syntax of asset files.

The Onyo working tree is not clean.
Changes not staged for commit: 
	 .onyo/templates/laptop.template
Please commit all changes or add new files to .gitignore
```

But I seem to get the same error with `onyo git status` and a few more commands. What should I check next?

**Comment by aqw** [comment]
You have found another bug in `onyo git`, that must have been introduced after Tobias filed this bug.

Onyo runs some checks before executing commands (similar to `onyo fsck`). One of these checks is ensuring that the git repo is clean.

You have created the template file, and thus there is an untracked file.

It seems this check is also done for `onyo git`, which somewhat defeats the purpose of the command.

**Comment by TobiasKadelka** [comment]
True. I wonder if it should be just simply removed, or if there would be value to run the checks in the end to get output describing the situation after the `onyo git` commands.

**Comment by aqw** [comment]
Closing as wont-fix with the merging of #201
